### PR TITLE
Ready code for 6.2 release and misc bug fixes

### PR DESCRIPTION
**Pull request by Christina-Kang**
Latest version of autorest requires that classes be initiated a little bit differently. This allows the CLI to properly consume the latest SDK.

Verifed:

- [ ] Build and test CI passes
- [ ] History and readme updated to reflect changes
- [ ] Package version updated according to semantic versioning rules
- [ ] Tests modified or added, when applicable
- [ ] Updated code owners file, when applicable
- [ ] Read the [PR checklist](https://github.com/Azure/service-fabric-cli/wiki/Checklist)

### CODE REVIEW

File: src/README.rst [hunk 1]
```diff
@@ -27,7 +27,9 @@ Unreleased
 - Add tests to verify correctness of help text (#71)
 - Add scaling policy parameter to the command for service create and the command for service update (#76)
 - Add new group called container with commands: invoke-api (invoke raw container REST API) and logs (get container logs) (#82)
-- Add support for passing json values to arguments as .txt files (#84)
+- Add support for passing json values to arguments as .txt files. Input starting with '@' are considered paths (#84)
+- Update to 6.2 Service Fabric runtime (#97)
```
Comment (samedder) [inline src/README.rst]: Can you change the name if that's the goal of this PR, `Reformat function calls` seems misleading :P

File: src/sfctl/custom_app.py [hunk 2]
```diff
@@ -166,7 +166,7 @@ def parse_app_params(formatted_params):
 
     res = []
     for k in formatted_params:
-        param = ApplicationParameter(k, formatted_params[k])
+        param = ApplicationParameter(key=k, value=formatted_params[k])
```
Comment (samedder) [inline src/sfctl/custom_app.py]: Why do we need to do this now? Did something change in the SDK so we have all these optional named arguments?

File: src/sfctl/helps/app.py [hunk 3]
```diff
@@ -56,11 +56,11 @@
...
       value of the parameters from the application manifest.
     parameters:
-        - name: --application-name
+        - name: --application-id
           type: string
-          short-summary: The name of the application
+          short-summary: The identity of the application.
           long-summary: "This is typically the full name of the application
-            with the 'fabric:' URI scheme. Starting from version 6.0,
+            without the 'fabric:' URI scheme. Starting from version 6.0,
```
Comment (samedder) [inline src/sfctl/helps/app.py]: Can you double check this changed in the SDK? This was a bug last time.

File: src/sfctl/custom_app.py (hunk 2 above)
Comment (Christina-Kang) [reply src/sfctl/custom_app.py]: Yes. The SDK was changed because of the new auto-rest

File: src/sfctl/helps/app.py (hunk 3 above)
Comment (Christina-Kang) [reply src/sfctl/helps/app.py]: I changed this because the URL it was producing is incorrect.

Comment (samedder) [reply src/sfctl/helps/app.py]: Can you test through the CLI, I found that the documentation was actually incorrect.

Comment (Christina-Kang) [reply src/sfctl/helps/app.py]: I have tested through the CLI just now. App id works :)

**Pull request merged**